docs/troubleshooting.md

```diff
@@ -0,0 +1,60 @@
+---  
+title: Troubleshooting
+has_children: false
+nav_order: 12
+---  
+
+
+# Index Not Found Exception / white.lock error while using a docker image
+
+Error message:
+
+ >  Error in custom provider, org.apache.lucene.index.IndexNotFoundException: no segments* file found in MMapDirectory@/app/data/odinson/index lockFactory=org.apache.lucene.store.NativeFSLockFactory@68e2d03e: files: [write.lock]
+
+Run the following command and index the documents using a docker image as described [here](docker.md)
+
+```bash
+rm -rf /path/to/data/odinson/index/
+```
+
+**NOTE**: Replace `/path/to/data/odinson` with the path to the directory containing `index`.
+
+
+# No such method error while using a docker image
+
+Error message:
+
+> 'void sun.misc.Unsafe.putInt(java.lang.Object, int, int)'
+
+Make sure you have java 11 installed (e.g., OpenJDK 11).
```

Comment:
This should only ever come up if you build an image locally without Java 11 installed.